spec/beaker-hiera/helpers_spec.rb

```diff
@@ -11,18 +11,8 @@ def logger
 end
 
 describe ClassMixedWithDSLHelpers do
-  let( :opts )   { Beaker::Options::Presets.env_vars }
-  let( :command ){ 'ls' }
-  let( :host )   { double.as_null_object }
-  let( :result ) { Beaker::Result.new( host, command ) }
-
-  let( :master ) { make_host( 'master',   :roles => %w( master agent default)    ) }
-  let( :agent )  { make_host( 'agent',    :roles => %w( agent )           ) }
-  let( :custom ) { make_host( 'custom',   :roles => %w( custom agent )    ) }
-  let( :dash )   { make_host( 'console',  :roles => %w( dashboard agent ) ) }
-  let( :db )     { make_host( 'db',       :roles => %w( database agent )  ) }
-  let( :hosts )  { [ master, agent, dash, db, custom ] }
-
+  let( :host  ) { make_host( 'master', :roles => %w( master agent default) ) }
+  let( :hosts ) { [ host ] }
```

Comment:
Not as far as I can see.  None of the tests are there to ensure that it either takes or returns an array of hosts, they just want to be sure to act on the default host.

I suspect that the array was just a holdover from copy-pasting from a more complex testing situation to get this one going.